src/violations.rs

```diff
@@ -3219,6 +3219,21 @@ impl AlwaysAutofixableViolation for DictGetWithDefault {
         DictGetWithDefault("var = dict.get(key, \"default\")".to_string())
     }
 }
+
+define_violation!(
+    pub struct UnpackInsteadOfConcatenatingToCollectionLiteral(pub String);
+);
+impl Violation for UnpackInsteadOfConcatenatingToCollectionLiteral {
+    fn message(&self) -> String {
+        let UnpackInsteadOfConcatenatingToCollectionLiteral(expr) = self;
+        format!("Consider `{expr}` instead of concatenation")
+    }
+
+    fn placeholder() -> Self {
```

Comment:
I believe we need to merge in `main`, then remove this method, and add `#[derive_message_formats]` just above `fn message`.

Reply:
Rebased!